src/components/ActionItems/SaveToPlaylist.jsx

```diff
@@ -0,0 +1,14 @@
+import { useData } from "../../contexts";
+
+export const SaveToPlaylist = () => {
+  const { setPlaylistModal } = useData();
+  return (
+    <span
+      className="option-item"
+      onClick={() => setPlaylistModal((isShow) => (isShow ? false : true))}
+    >
```

Comment:
Suggestion: You could instead return `!isShow` as follows
onClick={() => setPlaylistModal((isShow) => !isShow)}